src/app/service/photos/photos.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PhotosService {
+
+  constructor(private http: HttpClient) { }
+
+  getPhotos(): Promise<Photo[]> {
+    return this.http.get(`https://${environment.facebookphotosUrl}`, {
+      params: { token: environment.facebookphotosToken }
+    }).toPromise()
+    .then((response: any) =>
+    //is this the correct way to make an api call?
+      FB.api(
+        "/{photo-id}",
+        function (response) {
```

Comment:
Get used to arrow functions will help a lot.
In this case it will be 
```javascript
res => {
  if(res && !res.error) { }
}
```
or 
```javascript
(err, res) => {
  if ( res && !err ) { }
}
```

Reply:
I see, just googled it! Thanks for the information provided!